src/vellum/workflows/runner/runner.py

```diff
@@ -332,6 +332,18 @@ def initiate_node_streaming_output(output: BaseOutput) -> None:
                     parent=parent_context,
                 )
             )
+        except WorkflowInitializationException as e:
+            self._workflow_event_inner_queue.put(
+                NodeExecutionRejectedEvent(
+                    trace_id=node.state.meta.trace_id,
+                    span_id=span_id,
+                    body=NodeExecutionRejectedBody(
+                        node_definition=node.__class__,
+                        error=e.error,
+                    ),
+                    parent=parent_context,
+                )
+            )
```

Comment:
[oos] Is this for the case of subworkflows? I would actually expect the subworkflow nodes to catch and reraise as a node exception. Could do as a followup for now and depri